components/Artist.tsx

```diff
@@ -0,0 +1,77 @@
+import Picture from "../components/Picture";
+import styled from "styled-components";
+import { nanoid } from "nanoid";
+import { ReactPropTypes } from "react";
+
+//Typescript
+type ArtistProps = {
+  artistName: string;
+  location: string;
+  tattoos: string[];
+};
+//Typescript end
+
+export default function Artist({
+  artistName,
+  location,
+  tattoos,
+}: ArtistProps): JSX.Element {
+  return (
+    <StyledMain>
+      <StyledInfoCard>
+        Information
+        <li>
+          Artist:
+          <br /> {artistName}
```

Comment:
Try not to use `<br />`, use CSS properties or other tags to break the text. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br?retiredLocale=de#accessibility_concerns